diffa/lrc/src/account.cpp b/src/account.cpp
indea/lrc/x c677656..46acdac 100644
--- a/lrc/src/account.cpp
+++ a/lrc/src/account.cpp
@@ -1359,7 +1359,7 @@ bool Account::banCertificate(Certificate* c)
 }
 
 ///Ask the certificate owner (peer) to trust you
-bool Account::requestTrust( const URI& uri )
+bool Account::sendContactRequest( const URI& uri )
 {
    if (uri.isEmpty())
        return false;
@@ -1376,15 +1376,15 @@ bool Account::requestTrust( const URI& uri )
    return true;
 }
 
-bool Account::requestTrust(const ContactMethod* c)
+bool Account::sendContactRequest(const ContactMethod* c)
 {
     if (!c)
         return false;
 
-    return requestTrust(c->uri());
+    return sendContactRequest(c->uri());
 }
 
-bool Account::requestTrust( Certificate* c )
+bool Account::sendContactRequest( Certificate* c )
 {
    if ((!c) || (c->remoteId().isEmpty()))
       return false;
diffa/lrc/src/account.h b/src/account.h
indea/lrc/x 7fb92ce..96f7591 100644
--- a/lrc/src/account.h
+++ a/lrc/src/account.h
@@ -477,9 +477,9 @@ class LIB_EXPORT Account : public ItemBase {
       Q_INVOKABLE bool supportScheme   ( URI::SchemeType type ) const;
       Q_INVOKABLE bool allowCertificate( Certificate* c       )      ;
       Q_INVOKABLE bool banCertificate  ( Certificate* c       )      ;
-      Q_INVOKABLE bool requestTrust    ( Certificate* c       )      ;
-      Q_INVOKABLE bool requestTrust    ( const ContactMethod* c)     ;
-      Q_INVOKABLE bool requestTrust    ( const URI& uri       )      ;
+      Q_INVOKABLE bool sendContactRequest(Certificate* c);
+      Q_INVOKABLE bool sendContactRequest(const ContactMethod* c);
+      Q_INVOKABLE bool sendContactRequest(const URI& uri);
       Q_INVOKABLE QString accountDetail(const QString& param) const;
 
       //Setters
